.travis.yml

```diff
@@ -12,11 +11,13 @@ env:
   - VARNISH_VERSION=4.0
 
 matrix:
+  allow_failures:
+    # HHVM support for CURLOPT_PROTOCOLS was added recently but not yet
+    # available on Travis
+    - php: hhvm
   include:
-    - php: 5.5
-      env: VARNISH_VERSION=3.0
```

Comment:
Varnish 3.0 is still included: see https://github.com/FriendsOfSymfony/FOSHttpCache/pull/173/files#diff-354f30a63fb0907d4ad57269548329e3R20.